lib/libutils/isoc/arch/arm/sub.mk

```diff
@@ -7,7 +7,7 @@ srcs-$(CFG_ARM32_$(sm)) += arm32_aeabi_ldivmod_a32.S
 srcs-$(CFG_ARM32_$(sm)) += arm32_aeabi_ldivmod.c
 srcs-$(CFG_ARM32_$(sm)) += arm32_aeabi_shift.c
 
-ifeq ($(CFG_ULIBS_MCOUNT),y)
+ifeq ($(call cfg-one-enabled, CFG_ULIBS_MCOUNT CFG_FTRACE_SUPPORT),y)
```

Comment:
To be precise that's `CFG_SYSCALL_FTRACE` that should be used here instead of `CFG_FTRACE_SUPPORT`, because what this block needs to do is revert what is done by:
- [lib.mk](https://github.com/OP-TEE/optee_os/blob/6f48ab5d3e350bca562beced9e3a261f24f02396/mk/lib.mk#L22-L26): adding `-pg` to the user space build of libutils when `CFG_ULIBS_MCOUNT=y`
- [core.mk](https://github.com/OP-TEE/optee_os/blob/6f48ab5d3e350bca562beced9e3a261f24f02396/core/core.mk#L72-L74): adding `-pg` to the kernel space build of libutils when `CFG_SYSCALL_FTRACE=y`

That being said it would be much simpler to just remove `-pg` unconditionally since in fact we *never* want it. Especially since the test is also missing the kernel vs. user mode aspect. So I suggest simply dropping the `ifeq / endif` here.